_pages/policies/business-and-ops-policies/top-secret.md

```diff
@@ -6,27 +6,35 @@ tags:
 - top
 - security
 ---
+Most TTS employees hold moderate risk Public Trust positions. However, for certain projects, partners need TTS employees to have access to classified national security information. These employees need to be granted security clearance eligibility (a.k.a a security clearance) to be able to view this information. This page covers the basics like clearance levels, your clearance status, your clearance obligations, and clearance upgrade requests.
 
-Most TTS employees have a Public Trust (Moderate Risk - Level 5) clearance. However, for certain projects, clients need TTS employees to have a higher type of clearance so that they are able to view certain documents.  This is the process for TTS employees to request this clearance.
+## Clearance Levels
+There are [three national security clearance levels]((https://en.wikipedia.org/wiki/List_of_U.S._security_clearance_terms)): Confidential, Secret, and Top Secret. Work deemed Critical Sensitive requires a Top Secret clearance. Special Sensitive work requires access to Sensitive Compartmented Information and therefore a Top Secret / Sensitive Comparmented Information (TS/SCI) clearance. When TTS employees need a clearance, it is typically a TS/SCI.
```

Comment:
```suggestion
There are [three national security clearance levels](https://en.wikipedia.org/wiki/List_of_U.S._security_clearance_terms): Confidential, Secret, and Top Secret. Work deemed Critical Sensitive requires a Top Secret clearance. Special Sensitive work requires access to Sensitive Compartmented Information and therefore a Top Secret / Sensitive Compartmented Information (TS/SCI) clearance. When TTS employees need a clearance, it is typically a TS/SCI.
```

Reply:
the double-parens there were making the tests fail.

Reply:
good catch!